tool/travis.sh

```diff
@@ -7,36 +7,44 @@ set -e
 if [ -z "$PKG" ]; then
   echo -e '\033[31mPKG environment variable must be set!\033[0m'
   exit 1
-elif [ -z "$TASK" ]; then
-  echo -e '\033[31mTASK environment variable must be set!\033[0m'
-  exit 1
+fi
+
+if [ "$#" == "0" ]; then
+	echo -e "\033[31mExpected at least one task!\033[0m"
+	exit 1
 fi
 
 pushd $PKG
 pub upgrade
 
-case $TASK in
-dartanalyzer) echo
-  echo -e '\033[1mTASK: dartanalyzer\033[22m'
-  echo -e 'dartanalyzer --fatal-infos --fatal-warnings .'
-  dartanalyzer --fatal-infos --fatal-warnings .
-  ;;
-dartfmt) echo
-  echo -e '\033[1mTASK: dartfmt\033[22m'
-  echo -e 'dartfmt -n --set-exit-if-changed .'
-  dartfmt -n --set-exit-if-changed .
-  ;;
-test_0) echo
-  echo -e '\033[1mTASK: test_0\033[22m'
-  echo -e 'pub run test'
-  pub run test
-  ;;
-test_1) echo
-  echo -e '\033[1mTASK: test_1\033[22m'
-  echo -e 'pub run test --run-skipped'
-  pub run test --run-skipped
-  ;;
-*) echo -e "\033[31mNot expecting TASK '${TASK}'. Error!\033[0m"
-  exit 1
-  ;;
-esac
+while (( "$#" )); do
```

Comment:
huh?

Reply:
I added support for passing multiple tasks as arguments, this is "while there are more arguments" basically

Reply:
basically just lets you combine the analyzer/dartfmt tasks into one to save time

Reply:
Ah. So if we want to stack tasks together.

It's a trade off. Fewer things to startup is good.

Losing the clear signal that a failure was due to dartfmt or analyzer is a bit more annoying.

Idea: can we do fun bash tricks to run all tasks (even if one fails) so we get all of the results – and then do a `succeed if every subtask succeeds`.

I hate getting a dartfmt error. Fixing it. Only to find out there is an analyzer error, too...

Reply:
Of course this also leads me to think we should ponder creating a `.monorepo` config file or something so we can more clearly express this kind of intent without bending over backwards to create a valid `.travis.yml` file

Reply:
Ya I haven't thought to much about exactly how users would express grouping tasks like this - just playing around with it.

Reply:
> Idea: can we do fun bash tricks to run all tasks (even if one fails) so we get all of the results – and then do a succeed if every subtask succeeds.

I like that idea for this if we do decide its worth supporting in general (I think it probably is though)

Reply:
done https://travis-ci.org/dart-lang/build/jobs/336315880